sdcm/provision/common/utils.py

```diff
@@ -66,12 +66,38 @@ def configure_syslogng_target_script(host: str, port: int, throttle_per_second:
             # issue relate to shutdown - decided to ignore in scylladb/scylladb#15672
             not message(\\".*view update generator not plugged to push updates\\") and
 
+            # remove below workaround after dropping support for Scylla 2023.1 and 5.2 (see scylladb/scylla#13538)
```

Comment:
Please, use "TODO:" prefix for such cases.
It is very useful to find places where we should do something and distinguish it from the comments that just share the intentions and context.

Reply:
Added the TODO in the comment